### ISSUE TITLE
Grafana returns 401 on direct image render link from the image renderer

### ISSUE THREAD

**Issue by weedmaps-andychan** [grafana/grafana-image-renderer#91]
**What happened**:
Following the instruction on https://grafana.com/docs/grafana/latest/administration/image_rendering/#remote-rendering-service .  Image Renderer throws error on unauthorized access to grafana.

**What you expected to happen**: Image Renderer should be able to call grafana for direct rendered image

**How to reproduce it (as minimally and precisely as possible)**:
Following the instruction on https://grafana.com/docs/grafana/latest/administration/image_rendering/#remote-rendering-service

**Anything else we need to know?**:

**Environment**:
- Grafana Image Renderer version: latest
- Grafana version: 6.5.3
- Installed plugin or remote renderer service: As a service on a container
- OS Grafana Image Renderer is installed on: container
- User OS & Browser: Mac OS / Chrome
- Others:

```
1/21/2020 4:48:04 PM{"level":"info","message":"Render request received', 'url', http://grafana:3000/d-solo/N4gOslEWk/grafana-alert-troubleshooting?orgId=1&panelId=2&render=1"}
```

```
1/21/2020 4:48:04 PM{"0":"msg","1":"Failed to load resource: the server responded with a status of 401 (Unauthorized)","2":"url","3":"http://grafana:3000/d-solo/N4gOslEWk/grafana-alert-troubleshooting?orgId=1&panelId=2&render=1","4":"line","6":"column","level":"error","message":"Browser console error"}
```

**Comment by weedmaps-andychan** [comment]
I may be missing some details above.  It looks like this problem exists if I run more than 1 grafana container.  When grafana calls the renderer, it passes the renderKey and it looks like the renderer needs to callback to the original grafana within the cluster.  Am I correct?

**Comment by marefr** [comment]
@weedmaps-andychan that's correct.

**Comment by weedmaps-andychan** [comment]
The documentation is not too clear about that.  May be adding a few lines in README.

**Comment by DanCech** [comment]
When deploying multiple grafana webheads in a cluster you'll want to use a shared database.  If each one has a separate database you will have all sorts of problems.

**Comment by marefr** [comment]
The main issue right now is that Grafana stores the render key in memory and not in shared database and that is the reason renderer must callback to original Grafana.

**Comment by DanCech** [comment]
Aha, that is a problem then.

**Comment by marefr** [comment]
Yes, I definitely think so and should be quite straightforward to implement. Just need to create issue and prioritize it.

**Comment by erkiesken** [comment]
We also ran into this, there's a Grafana deployment with 2 pods in k8s for us, and Grafana renderer as separate deployment. And GF_RENDERING_CALLBACK_URL is using a k8s Service, so might hit one of the 2 Grafana pods, sometimes failing sometimes succeeding. Would be nice to have support for this setup too.

**Comment by marefr** [comment]
Closed by https://github.com/grafana/grafana/pull/22031 and will be included in Grafana v6.7.0.
